ocs_ci/ocs/ocp.py

```diff
@@ -999,6 +999,23 @@ def get_all_resource_names_of_a_kind(kind):
     ]
 
 
+def get_all_resource_of_kind_containing_string(search_string, kind):
+    """
+    Return all the resource of kind which name contain search_string
```

Comment:
Add 1 empty line in between each docstring section. 
See https://ocs-ci.readthedocs.io/en/latest/coding_guidelines.html?highlight=docstring#coding-guidelines-for-ocs-ci-project
Please apply in all other functions in this PR